ra/ra.go

```diff
@@ -1748,3 +1756,14 @@ func (ra *RegistrationAuthorityImpl) createPendingAuthz(ctx context.Context, reg
 	authz.Combinations = comboBytes
 	return authz, nil
 }
+
+// validChallengeStillGood checks whether the valid challenge in an authorization uses a type
+// which is still enabled
+func (ra *RegistrationAuthorityImpl) validChallengeStillGood(authz *core.Authorization) bool {
```

Comment:
Nit: `authzValidChallengeEnabled` might be a more descriptive name for this func/rpc